platform/source/lib/ax/platform/rest.py

```diff
@@ -37,7 +37,7 @@
 from ax.platform.deployments import Deployment
 from ax.platform.axmon_main import AXMon
 from ax.platform.cluster_config import AXClusterConfig, SpotInstanceOption
-from ax.platform.minion_management.spot_instance_option_manager import SpotInstanceOptionManager
+from ax.platform.minion_manager.spot_instance_option_manager import SpotInstanceOptionManager
```

Comment:
NIT: since spot_instance_option_manager file is the internal of package ax.platform.minion_manager, you can add `from .spot_instance_option_manager import SpotInstanceOptionManager` in the __init__.py file of the package, then you can just do `from ax.platform.minion_manager import SpotInstanceOptionManager`, without knowing details about internal file structure of the package.

Reply:
Done.